.travis.yml

```diff
@@ -1,74 +1,21 @@
-# Generic MoveIt Travis Continuous Integration Configuration File
-# Works with all MoveIt! repositories/branches
-# Author: Dave Coleman, Jonathan Bohren, Robert Haschke
+# This config file for Travis CI utilizes https://github.com/ros-planning/moveit_ci/ package.
```

Comment:
as @v4hn pointed out, would be better if this was in a separate PR and merged in first